app/src/main/java/com/alphawallet/app/repository/TransactionRepository.java

```diff
@@ -23,6 +29,7 @@
 import io.reactivex.Single;
 import io.reactivex.schedulers.Schedulers;
 import io.realm.Realm;
+import timber.log.Timber;
 
 public class TransactionRepository implements TransactionRepositoryType {
 
```

Comment:
Can the changes to this file be reverted? There doesn't appear to be any change here.

Reply:
Done